### ISSUE TITLE
IQB Tastatur am PC

### ISSUE THREAD

**Issue by helfjama** [iqb-berlin/verona-modules-aspect#537]
Die IQB Tastatur öffnet sich auch am PC bis man eine Eingabe betätigt hat. 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaG1yIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--1614e0e43058cc585873a13a74628517b38a3ba8/grafik.png)
Danach öffnet sie sich wie gewünscht nicht mehr.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaHVyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--cb6647ce2fe335b9622ab0a70699284324c0e6b6/grafik.png)
Dieser Fehler wurde von uns erst am 18.09. um 13 Uhr bemerkt, wir sind uns eigentlich auch relativ sicher, dass er noch nicht länger existiert.

**Comment by helfjama** [comment]
**Ergänzung:** Das Verhalten ist dasselbe wie an Tablets mit einer externen Tastatur - sobald diese das erste Mal verwendet wird, verschwindet die IQB Tastatur. Wird der Browser (Firefox, Version von 12.09.) vielleicht einfach als mobiler Browser erkannt?

**Comment by jkoelm** [comment]
Einen Tag später können wir das Problem nicht mehr reproduzieren, daher liegt das Ricket erst mal auf Eis, bis wir es sinnvoll reproduzieren können.

**Comment by jojohoch** [comment]
> **:** Das Verhalten ist dasselbe wie an Tablets mit einer externen Tastatur - sobald diese das erste Mal verwendet wird,

Das ist genau das gewünschte Verhalten: Der Browser kann nicht erkennen, ob am iPad eine externe Taststatur angeschlossen ist. Die Tastatatur wird aber geöffnet, da es sich um ein mobiles Gerät handelt. Erfolgt aber dann eine Abgabe über die Tastatur, wird ein weiteres Öffnen der IQB-Tastatur unterbunden.

**Comment by helfjama** [comment]
@jojohoch Ja, auf Tablets mit externer Tastatur ist das das gewünschte Verhalten, aber eben nicht auf nicht mobilen Geräten. Dort hat es zurecht einige Reviewer irritiert.

**Comment by helfjama** [comment]
Zurückstellen bis nach BiSta Pilot, dann nochmal genauer testen
Außerdem Idee: Diese Abfrage Unitübergreifend in Verona-Schnittstelle?
